src/components/NewWorkspaceModal.ts

```diff
@@ -319,11 +319,19 @@ const NewWorkspaceModal = withDisplayName(
     };
 
     const isBillingProjectApplicable = (project: BillingProject): boolean => {
-      // As of AJ-1164, if requireEnhancedBucketLogging is true, then azure billing projects are ineligible.
-      // This coupling of enhanced bucket logging and billing project may change in the future.
-      if (!!workflowImport || !!requireEnhancedBucketLogging) {
+      // This is used when importing data to enforce a specific cloud.
+      if (cloudPlatform && project.cloudPlatform !== cloudPlatform) {
+        return false;
+      }
+      if (workflowImport) {
```

Comment:
Lint suggested I remove the "!!" which I agree with given that workflowImport is typed as a boolean.